### ISSUE TITLE
Module settings. Impossible to save an empty token.

### ISSUE THREAD

**Issue by cheelahim** [craftyclicks/magento2#6]
It is impossible to save empty token in the module settings.
Platform: Magento 2.0.5 EE
### Steps to reproduce:
1. Go to Stores->Configuration->Crafty Clicks->Click to Address
2. Enter non-empty Access Token and save the config
3. Clear Access Token input and save the config.
### Expected result:

New empty token value is saved
### Actual result:

Access token value is not changed. Old value is used.

**Comment by gs-mancs** [comment]
TL;DR: the official Magento EE package is broken, please don't use it.
This is not the first time that there's a significant issue with the downloadable pack.
Just stick to the git version and use composer yourself.

So, this works perfectly on the latest CE & EE versions.
Except on EE downloadable package.

File that causes this issue is 
app/code/Magento/Config/Model/Config/Backend/Encrypted.php
NOTE: on the downloadable package, it will be under Vendor.

There's a little bit of code difference between the EE package version, and the Git CE version.

The EE files do not include this specific file, and although in the package it should perfectly match up to the CE edition's version, it doesn't.
This code was added on 2016/02/05.
https://github.com/magento/magento2/commit/c41b4fee66b1b1dacdc5dd5e1109741a51b2c213

I left a comment for this line & I'll probably chase some Magento people to get this included into the official package..
If you're stuck with the downloadable zip version (for whatever reason) don't expect a fix before 2.0.7

As this is **NOT** an issue with the module itself, no special fixes will be deployed from our side.

PS: There's literally nothing I can do, Magento people _insisted_ that I use the encrypted config model for the access token on a previous partner review.
